### PR TITLE
image-source: Add psd and *.* to file filter.

### PR DESCRIPTION
**Pull request by pkviet**
Add psd (photoshop) to file filter and more generally enable display of all files (\*.\*).

### CODE REVIEW

File: plugins/image-source/image-source.c [hunk 1]
```diff
@@ -214,12 +214,13 @@ static void image_source_tick(void *data, float seconds)
 
 
 static const char *image_filter =
-	"All formats (*.bmp *.tga *.png *.jpeg *.jpg *.gif);;"
```
[the remark this answers is not in the document]
Comment (pkviet) [reply plugins/image-source/image-source.c]: Sure, I've added back the line and restored original order.

**Comment by pkviet** [comment]
@RytoEX good point; i've implemented it and removed the period.

File: plugins/image-source/image-source.c [hunk 2]
```diff
@@ -214,12 +214,14 @@ static void image_source_tick(void *data, float seconds)
...
+	"All formats (*.bmp *.tga *.png *.jpeg *.jpg *.gif *.psd);;"
 	"BMP Files (*.bmp);;"
 	"Targa Files (*.tga);;"
 	"PNG Files (*.png);;"
 	"JPEG Files (*.jpeg *.jpg);;"
-	"GIF Files (*.gif)";
+	"GIF Files (*.gif);;"
+	"Photoshop Files (*.psd);;"
```
Comment (RytoEX) [inline plugins/image-source/image-source.c]: Not sure if it makes more sense to say "Photoshop Files" or "PSD Files".  The only item in the list that doesn't follow the pattern of "\<Extension> Files" is Targa.  As an aside or historical note, TARGA is apparently an acronym.

Comment (pkviet) [reply plugins/image-source/image-source.c]: ok changed photoshop to psd

**Pull request merged**